pep-513.rst

```diff
@@ -9,7 +9,7 @@ Status: Draft
 Type: Informational
 Content-Type: text/x-rst
 Created: 19-Jan-2016
-Post-History: 19-Jan-2016
+Post-History: 24g-Jan-2016
```

Comment:
Stray `g`. Also, IIRC this is supposed to be a list of dates on which a draft was posted to a mailing list (I guess to help people trying to track down the discussion archives later)?

On which note I guess it might be a good idea to include a link somewhere in the references to the manylinux-discuss archives where we debated stuff initially.